### DISCUSSION TITLE
can owner of one organisation see other organisations if he's not added

### DISCUSSION THREAD

**Discussion by candormy** [community/community#31701] [New to GitHub]
Hi 

I have created two organization and one of them I gave ownership to the developer to manage it. However, I have other two orgs that this developer is not part of. My question is, can this developer be able to see my other orgs as well?

**Comment by jfullstackdev** [comment]
in my experience, there is no access if you did not add him directly, or else that will be a security bug because all connected users can access Orgs that they are not part of
